robot/impl/local_robot_test.go

```diff
@@ -2181,6 +2181,34 @@ func TestResourcelessModuleRemove(t *testing.T) {
 	})
 }
 
+func TestKill(t *testing.T) {
+	// this test will not pass in CI as the managed process's manage goroutine
+	// will not return from Wait() and thus fail the goroutine leak detection.
+	t.Skip()
```

Comment:
I appreciate that this test exists, but is it even meaningful to have it around? A `t.Skip` with no TODO to unskip feels wrong to me. Or were you meaning to remove this `t.Skip`?

Reply:
it's meaningful to be able to run locally, but added a ticket as well https://viam.atlassian.net/browse/RSDK-9722